lib/ExpandableListItem/ExpandableListItem.js

```diff
@@ -289,6 +290,19 @@ module.exports = kind(
 			this.bubble('onRequestScrollIntoView');
 		}
 	},
+	
+	/**
+	* Check for the first item in the client area, and close the list
+	* if the focus moves out of the drawer and auto-collapse is true
+	*
+	* @private
+	*/
+	spotlightUp: function(inSender, inEvent) {
+		var children = Spotlight.getChildren(this.$.client);
+		if (children.length && inEvent.originator == children[0] && this.getAutoCollapse() && this.getOpen()) {
```

Comment:
@rajyavardhanp Do you have a reason for checking the event comes from first spotlight item?
If there is no condition {{children.length && inEvent.originator == children[0]}}, are there specific problem?